axelrod/tournament_manager_factory.py

```diff
@@ -1,6 +1,10 @@
-from tournament_manager import *
+from __future__ import absolute_import
+from collections import OrderedDict
+
 import axelrod.strategies
 
+from .tournament_manager import *
```

Comment:
This is something I don't know about python3, why the `.tourna...`, why the `.`?